token-lending/program/src/state/lending_market.rs

```diff
@@ -66,22 +67,22 @@ impl Pack for LendingMarket {
     fn pack_into_slice(&self, output: &mut [u8]) {
         let output = array_mut_ref![output, 0, LENDING_MARKET_LEN];
         #[allow(clippy::ptr_offset_with_cast)]
-        let (version, bump_seed, owner, quote_token_mint, token_program_id, _padding) =
-            mut_array_refs![output, 1, 1, PUBKEY_BYTES, PUBKEY_BYTES, PUBKEY_BYTES, 128];
+        let (version, bump_seed, owner, quote_currency, token_program_id, _padding) =
+            mut_array_refs![output, 1, 1, PUBKEY_BYTES, 32, PUBKEY_BYTES, 128];
```

Comment:
This is to allow the implementation to be expanded later with backwards compatibility.